src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        printLogo();
+        greetUser();
+
+        String userLineInput = "initialize";
+        Task[] taskItems = new Task[100]; //create array to store text entered by user
+        int taskCounter = 0; //track number of tasks
+
+        Scanner in = new Scanner(System.in); //Create scanner object to take in input
+
+        while (!userLineInput.equals("bye")) {
+            userLineInput = in.nextLine(); //user input
+            String[] userWords = userLineInput.split(" ");
+
+            printDottedLine();
+            switch (userWords[0]) {
+            case "list":
+                userInputIsList(taskItems, taskCounter);
+                continue;
+
+            case "todo":
+                userInputIsTodo(userLineInput, taskItems, taskCounter);
+                taskCounter++;
+                printTaskCount(taskCounter);
+                printDottedLine();
+                continue;
+
+            case "event":
+                userInputIsEvent(userLineInput, taskItems, taskCounter);
+                taskCounter++;
+                printTaskCount(taskCounter);
+                printDottedLine();
+                continue;
+
+            case "deadline":
+                userInputIsDeadline(userLineInput, taskItems, taskCounter);
+                taskCounter++;
+                printTaskCount(taskCounter);
+                printDottedLine();
+                continue;
+
+
+            case "done":
+                userInputIsDone(taskItems, taskCounter, userWords[1]);
+                continue;
+
+            case "bye":
+                userInputIsBye();
+                break;
+
+            default:
+            }
+        }
+    }
+
+    private static void userInputIsTodo(String userLineInput, Task[] taskItems, int taskCounter) {
+        int todoInputLength = userLineInput.length();
+        String todoTaskInput = userLineInput.substring(4, todoInputLength);
+        taskItems[taskCounter] = new ToDo(todoTaskInput);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(taskItems[taskCounter].description);
+    }
+
+    private static void userInputIsDeadline(String userLineInput, Task[] taskItems, int taskCounter) {
+        int byPosition = userLineInput.indexOf("/by");
+        int deadlineInputLength = userLineInput.length();
+        String deadlineTaskInput = userLineInput.substring(8, byPosition - 1);
+        String deadlineDateInput = userLineInput.substring((byPosition + 4), deadlineInputLength);
```

Comment:
Avoid the use of magic numbers such as "8" as well. Consider extracting them out into constants.